packages/user/src/components/DropdownUserMenu.tsx

```diff
@@ -49,9 +46,7 @@ const DropdownUserMenu: React.FC<Properties> = ({ userMenuList }) => {
 
   const fallbackItems = [profileRoute, signoutRoute];
 
-  const menuItems = userMenuList
-    ? [...userMenuList, ...fallbackItems]
-    : fallbackItems;
+  const menuItems = userMenu ? [...userMenu, ...fallbackItems] : fallbackItems;
```

Comment:
Should the userMenu override the fallbackItems? From the name, fallbackItems are supposed to be used only when its not provided in userMenu, right? If yes, then it should be `[...fallbackItems, ...userMenu]`

Reply:
it should be but we are extending it in the app from very beginning. so fi we are to refactor this it would be better to do it in separate ticket.